app.rb

```diff
@@ -97,10 +97,9 @@ def check_spotify(on_spotify)
   end
 
   def save_data
-    puts @music_album
-    puts @genres
-    all_data = [@genres, @music_album]
-    file_paths = ['./genre.json', './music_album.json', './module/rentals.json']
+    puts 'Data saved successfully'
```

Comment:
I'll suggest you put this line at the end of the method, so that when the an error occurs it doesn't display data saved successfully.